### PR TITLE
Add HotSync administrator manual

### PR DESCRIPTION
**Pull request by Stell0**
Nethserver/dev#5401

### CODE REVIEW

File: administrator-manual/en/hotsync.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
+.. _hotsync-section:
+
+=======
+HotSync
+=======
+
+HotSync aims to reduce downtime in case of failure, syncing your |product| with another one, that will be manually activated in case of master server failure
```
Comment (alepolidori) [inline administrator-manual/en/hotsync.rst]: please inset a final dot

File: administrator-manual/en/hotsync.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Normally, when a hardware damage occurs, the time needed to restore service is:
```
Comment (alepolidori) [inline administrator-manual/en/hotsync.rst]: "when an hardware" ?

File: administrator-manual/en/hotsync.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+By default all data included in backup are synchronized every 15 minutes. MariaDB databases are synchronized too, unless databases synchronization isn't disabled.
+Applications that use PostgreSQL are synchronized (Mattermost, Webtop5) unless databases synchronization isn't disabled.
+
+
+Terminology
+===========
+
+- MASTER is the production system SLAVE is the spare server
+- SLAVE is switched on, with an IP address different than MASTER
+- Every N minutes, MASTER makes a backup on SLAVE
```
Comment (alepolidori) [inline administrator-manual/en/hotsync.rst]: is it better to specify how many minutes ?

File: administrator-manual/en/hotsync.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+    [root@slave]# config setprop rsyncd password <PASSWORD>
+    [root@slave]# config setprop hotsync role slave
+    [root@slave]# config setprop hotsync MasterHost <MASTER_IP>
+    [root@slave]# signal-event nethserver-hotsync-update
+
+
+<PASSWORD> must be the same on master and slave.
+
+If mysql or postgresql are installed, they will be synced by default. To disable databases sync
```
Comment (alepolidori) [inline administrator-manual/en/hotsync.rst]: -> "they will be synchronized"

File: administrator-manual/en/hotsync.rst (hunk 2 above)
Comment (Stell0) [reply administrator-manual/en/hotsync.rst]: nope https://www.a-or-an.com/a_an/hardware

File: administrator-manual/en/hotsync.rst (hunk 1 above)
Comment (Stell0) [reply administrator-manual/en/hotsync.rst]: done 👍

**Comment by DavidePrincipi** [comment]
Added Transifex configuration: e41a48ea62b23705f21e2272d854bbe617b745bb

**Pull request merged**